src/System.Private.ServiceModel/src/System/IdentityModel/SecurityMessageSerializationException.cs

```diff
@@ -21,5 +23,10 @@ public SecurityMessageSerializationException(String message, Exception innerExce
             : base(message, innerException)
         {
         }
+
+        protected SecurityMessageSerializationException(System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context)
```

Comment:
NIT: No need `System.Runtime.Serialization` since you have `using` statement already.